code/02_create-figures.R

```diff
@@ -109,16 +111,16 @@ fig2 <-
         axis.ticks = element_blank(),
         legend.title = element_blank()) +
   annotate("text", 
-           x = 4e+05, y = 620000, 
+           x = 4e+05, y = 630000, 
            label = paste0("NHS ", unique(fig2$HBName[which.min(fig2$qom)]),
                           ": ", min(round_half_up(fig2$qom, 1)), "%"), 
-           size = 3.5,
+           size = 2,
            fontface = 2) +
   annotate("text", 
            x = 4e+05, y = 1100000, 
            label = paste0("NHS ", unique(fig2$HBName[which.max(fig2$qom)]),
                           ": ", max(round_half_up(fig2$qom, 1)), "%"), 
-           size = 3.5,
+           size = 2,
            fontface = 2)
 
 ggsave(here("markdown", "figures", "figure-2.png"), 
```

Comment:
I think the current width and height make the annotation too small after embedding into the report. After testing can we set the width = 13 and height = 14? I think that looks better.

Reply:
Thank Tina - I've tweaked size and increased font size to 2.5 and looks better now. I've also added the following bit of code to the word macro to centre all images. 

For Each oILShp In ActiveDocument.InlineShapes
oILShp.Select
Selection.ParagraphFormat.Alignment = wdAlignParagraphCenter
Next